pom.xml

```diff
@@ -343,18 +343,6 @@
               </goals>
             </execution>
             <execution>
-              <id>HBaseManagedTimeTests</id>
```

Comment:
Oh this is not even being used

Reply:
Must have been copied from HBase, then lurked here for years...

Reply:
Looking at again, this still might have effect on Pherf.
It probably should be purged, but not today.